aspics/params.py

```diff
@@ -130,23 +176,26 @@ def asarray(self):
                     ],
                     dtype=np.float32,
                 ),
+                self.health_risk_multipliers,
+                self.bmi_multipliers,
+                self.sex_multipliers,
+                self.ethnicity_multipliers,
             ]
         )
 
     @classmethod
     def fromarray(cls, params_array):
         location_hazard_multipliers = LocationHazardMultipliers(
             retail=params_array[8],
-            nightclubs=params_array[9],
-            primary_school=params_array[10],
-            secondary_school=params_array[11],
-            home=params_array[12],
-            work=params_array[13],
+            primary_school=params_array[9],
```

Comment:
Haha, this is all awful. :D  Manually flattening all of the parameters and doing the unpacking here and needing to know if the indices are correct.

Since you got this working for now, let's go with it. I'll look into other ways we can do this.